lib/Auth/Process/PrivacyideaAuthProc.php

```diff
@@ -204,7 +212,7 @@ private function enrollToken(string $stateID, string $username): string
             $type = $this->authProcConfig['typeOfTokenToEnroll'];
             $description = "Enrolled with simpleSAMLphp";
 
-            $response = $this->pi->enrollToken($username, $genkey, $type, $description);
+            $response = $this->pi->enrollToken($username, $genkey, $type, $description, $headersToForward);
```

Comment:
rename $headersToForward to $headers

Reply:
fixed [f2bb187](https://github.com/privacyidea/simplesamlphp-module-privacyidea/pull/198/commits/f2bb187f584e1a7a8c5c5fec38dee8d19a3e6b78)